conf/allofus.config

```diff
@@ -0,0 +1,23 @@
+process {
+    memory = { 6.GB * task.attempt }
```

Comment:
The only thing is the resource allocation needs to be slightly different on the AOU RW, given that they have couple memory-CPU constraints. Which is why these had to be changed here when you run on the AOU RW. I can update the max_retries, but there are additional error codes that need to be added when running on google life sciences API.

Reply:
Right I remember, what is the exact coupling, is it a factor? If this ie the case we could probably let the user chose memory only and infer cpu based on it, e.g. cpu = memory / 4 or something similar?

Reply:
It seems to be about 6 GB per 1 cpu. So we would have to make the memory a function of the number of CPUs. 

For example:

cpu = 1 * task.attempt
memory = 6.GB * cpu

Or something like this?

Reply:
Additionally - When CPUs requested is > 2, you must request an even number of cores. Ie. 4, 6, 8 etc. Another odd feature that I found when running the end-to-end test.

Reply:
I guess if you wanted the user to choose memory, you could just do the inverse operation. But CPUs would have to be an integer, so something like:

cpu = ceiling(memory/6)

Unsure exactly how this can be implemented at the nextflow configuration level. Would probably have to use the Math() operators.

Reply:
> It seems to be about 6 GB per 1 cpu. So we would have to make the memory a function of the number of CPUs.
> 
> For example:
> 
> cpu = 1 * task.attempt memory = 6.GB * cpu
> 
> Or something like this?

Then the multithreaded label does not respect this factor does it? I see 8GB for 2 cpus, should it not be 12GB? Any chance you can this info from all of us support team?

Reply:
> I guess if you wanted the user to choose memory, you could just do the inverse operation. But CPUs would have to be an integer, so something like:
> 
> cpu = ceiling(memory/6)
> 
> Unsure exactly how this can be implemented at the nextflow configuration level. Would probably have to use the Math() operators.

fair enough!

Reply:
see [5528ec3](https://github.com/TARGENE/targene-pipeline/pull/174/commits/5528ec36d9988b6a9506a9c294d0baf6568e50aa)